Owski Hacker pcb Buisness card

owski9 opened the pull request:
## Submission Checklist:

<!--To check the checkboxes, simply replace the empty space in the brackets with a captial X, like this: [ ] becomes [X]-->

- [X] I am a current high school, middle school, or home schooled student.
  - [X] I am 18 or under
  - [X] I have filled out the [verification form](https://airtable.com/app4Bs8Tjwvk5qcD4/pagxECjJZOgvKVnLd/form)
- [X] I have joined the [`#electronics`](https://hackclub.slack.com/archives/C056AMWSFKJ) channel on [Slack](https://hackclub.com/slack/?event=onboard)
- [X] I made this board from scratch, even if I followed a tutorial
- [X] I have followed the directions in [README.md](https://github.com/hackclub/OnBoard/blob/main/README.md#getting-started)
  - [X] Created a folder under `onboard/projects`
  - [X] Filled out `TEMPLATE.md` as `README.md` inside your project folder
  - [Z] Uploaded `gerber.zip` and `schematic.pdf` inside your project folder
  - [X] Created a folder called `src` and uploaded design files
  - [X] Uploaded screenshot of PCB order called `cart.png` with all costs included
  - [X] I am ordering the smallest batch size (ie. JLCPCB's smallest order is for 5 boards)
  - [X] If outside the US I've checked that I can afford the customs charges in my country (which isn't covered by the grant)

- [X] (Optional) This project is from a tutorial[^1].
- [ ] (Optional) This project is for OnBoard Live[^2].
- [x] (Optional) I'm in a FIRST (FRC, FTC, FLL, etc.) team. The number is: ____

[^1]: Projects from a tutorial are 100% fine! We just want to ask so we can count how people are using tutorials.
[^2]: OnBoard Live is a special version of OnBoard where you can earn more money for designing advanced boards. Check out the #onboard-live channel on [our Slack](https://hackclub.com/slack/?event=onboard)!

<!-- -Submission- -->

prophetorpheus commented:
<!-- MY-ONBOARD-BOT --> Hi, I'm Orpheus Leap! Here to help you review your PR.
  
**projects/OWSKI_PCB_HACKER CARD:**

Required files
||||
|---|---|---|
|✅| README.md | A description of your project |
|❌ | cart.png | You need to include a screenshot of your JLCPCB. Check out [these instructions](https://github.com/hackclub/OnBoard/blob/main/docs/ordering_from_JLCPCB.md#pcb-review). If you already have one, make sure it's a PNG file named exactly "cart.png". |
|✅| gerber.zip | |
|✅ | [schematic.pdf](<https://raw.githubusercontent.com/hackclub/OnBoard/6d35fc2cd4a29ee224bcccf87826035b0cf4d495/projects/OWSKI_PCB_HACKER CARD/schematic.pdf>) | Manually check [schematic.pdf](<https://raw.githubusercontent.com/hackclub/OnBoard/6d35fc2cd4a29ee224bcccf87826035b0cf4d495/projects/OWSKI_PCB_HACKER CARD/schematic.pdf>) |
|✅| Source files - EasyEDA |  Found both PCB and SCH files |

You can view a render of your board over on [gerber.zip/2d](<https://gerber.zip/2d/?mode=layers&boardUrl=https://raw.githubusercontent.com/hackclub/OnBoard/6d35fc2cd4a29ee224bcccf87826035b0cf4d495/projects/OWSKI_PCB_HACKER CARD/gerber.zip>)!
		
  Happy OnBoarding!
 <sub>This bot is a simpler helper for common submission types. If there's an error with this, please point it out and someone from the OnBoard team will look at it manually.</sub>

owski9 commented:
I accidentally marked a box with [Z] instead of X, but I did mean to mark it with [X]

owski9 commented:
> Hi, I'm Orpheus Leap! Here to help you review your PR.
> **projects/OWSKI_PCB_HACKER CARD:**
> 
> Required files
> 
> ✅	README.md	A description of your project
> ❌	cart.png	You need to include a screenshot of your JLCPCB. Check out [these instructions](https://github.com/hackclub/OnBoard/blob/main/docs/ordering_from_JLCPCB.md#pcb-review). If you already have one, make sure it's a PNG file named exactly "cart.png".
> ✅	gerber.zip	
> ✅	[schematic.pdf](https://raw.githubusercontent.com/hackclub/OnBoard/6d35fc2cd4a29ee224bcccf87826035b0cf4d495/projects/OWSKI_PCB_HACKER%20CARD/schematic.pdf)	Manually check [schematic.pdf](https://raw.githubusercontent.com/hackclub/OnBoard/6d35fc2cd4a29ee224bcccf87826035b0cf4d495/projects/OWSKI_PCB_HACKER%20CARD/schematic.pdf)
> ✅	Source files - EasyEDA	Found both PCB and SCH files
> You can view a render of your board over on [gerber.zip/2d](https://gerber.zip/2d/?mode=layers&boardUrl=https://raw.githubusercontent.com/hackclub/OnBoard/6d35fc2cd4a29ee224bcccf87826035b0cf4d495/projects/OWSKI_PCB_HACKER%20CARD/gerber.zip)!
> 
> Happy OnBoarding! This bot is a simpler helper for common submission types. If there's an error with this, please point it out and someone from the OnBoard team will look at it manually.



> Hi, I'm Orpheus Leap! Here to help you review your PR.
> **projects/OWSKI_PCB_HACKER CARD:**
> 
> Required files
> 
> ✅	README.md	A description of your project
> ❌	cart.png	You need to include a screenshot of your JLCPCB. Check out [these instructions](https://github.com/hackclub/OnBoard/blob/main/docs/ordering_from_JLCPCB.md#pcb-review). If you already have one, make sure it's a PNG file named exactly "cart.png".
> ✅	gerber.zip	
> ✅	[schematic.pdf](https://raw.githubusercontent.com/hackclub/OnBoard/6d35fc2cd4a29ee224bcccf87826035b0cf4d495/projects/OWSKI_PCB_HACKER%20CARD/schematic.pdf)	Manually check [schematic.pdf](https://raw.githubusercontent.com/hackclub/OnBoard/6d35fc2cd4a29ee224bcccf87826035b0cf4d495/projects/OWSKI_PCB_HACKER%20CARD/schematic.pdf)
> ✅	Source files - EasyEDA	Found both PCB and SCH files
> You can view a render of your board over on [gerber.zip/2d](https://gerber.zip/2d/?mode=layers&boardUrl=https://raw.githubusercontent.com/hackclub/OnBoard/6d35fc2cd4a29ee224bcccf87826035b0cf4d495/projects/OWSKI_PCB_HACKER%20CARD/gerber.zip)!
> 
> Happy OnBoarding! This bot is a simpler helper for common submission types. If there's an error with this, please point it out and someone from the OnBoard team will look at it manually.

I have added cart.png?

blazecoding2009 reviewed (changes requested):
Hey @owski9 

Please upload a proper schematic.pdf and not the PCB view

owski9 commented:
> Hey @owski9
> 
> Please upload a proper schematic.pdf and not the PCB view

I just fixed it I think. Please reply if its not working. In my fork I added the proper schematic.pdf, I'm not sure if it automatically updated the PR but my fork is correct and It didn't give me an option to open a new PR.

blazecoding2009 reviewed (approved):
Looks good! have fun with your [grant](https://hcb.hackclub.com/grants/XyWIjl)
